App/components/Checkbox/index.ts

```diff
@@ -0,0 +1 @@
+export { default as Checkbox } from './checkbox'
```

Comment:
change to `export { Checkbox } from './checkbox'`. And should we export props like in other components?